01-backup.md

```diff
@@ -803,6 +803,31 @@ $ git status --ignored
 nothing to commit, working directory clean
 ~~~
 
+> ## Committing Changes to Git {.challenge}
+> Which command(s) below would save changes of `myfile.txt` to my local git repository?
+>
+> 1. `git commit -m "my recent changes"`
+> 2. `git init myfile.txt`
+>    `git commit -m "my recent changes"`
+> 3. `git add myfile.txt`
+>    `git commit -m "my recent changes"`
+> 4. `git commit -m myfile.txt "my recent changes"`
+
+> ## Recovering Older Versions of a File {.challenge}
+> Jennifer has made changes to the Python script that she has been working on for weeks, and the 
+> modifications she made this morning "broke" the script and it no longer runs. She has spent
+> ~ 1hr trying to fix it, with no luck...
+>
+> Luckily, she has been keeping track of her revisions using git! Which commands below will 
+> let her recover the last committed (and working) version of her Python script called
```

Comment:
Sure, I only added that for dramatic emphasis, it is not necessary and I can see how it could be potentially distracting.